### PR TITLE
RFC: pre-proposal for blueprint store

### PR DESCRIPTION
**Pull request by emilk**
Based on a discussion today

### Checklist
* [x] I have read and agree to [Contributor Guide](https://github.com/rerun-io/rerun/blob/main/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/rerun-io/rerun/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: design/blueprint_store.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+
+### UI components
+The ui components are quite specific for the type of blueprint. Here are a few example:
+
+* `root`:
+    * `TimeControl` (globally selected timeline, time, play state, etc)
+* Top bar:
+    * `visible`
+* Blueprint panel:
```
Comment (nikolausWest) [inline design/blueprint_store.md:56]:
```suggestion
* Blueprint view:
```

File: design/blueprint_store.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    * `visible`
+    * `width`
+* Selection panel:
```
Comment (nikolausWest) [inline design/blueprint_store.md:59]:
```suggestion
* Selection view:
```

File: design/blueprint_store.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    * `visible`
+    * `width`
+* Time panel:
```
Comment (nikolausWest) [inline design/blueprint_store.md:62]:
```suggestion
* Streams view:
```

Comment (emilk) [reply design/blueprint_store.md:62]: no time, only streams

**Review by Wumpf — APPROVED** [review]
lgtm! Feels though like it's missing many details on the stuff that's further out on entity-path connection/interaction and the default/override mechanism, why we ended up there etc.

**Pull request merged**